### PR TITLE
DDF-4101 Enabled persistent notifications by default

### PR DESCRIPTION
**Pull request by AzGoalie**
#### What does this PR do?
Enables persistent notifications by default and changes expiration date from 1 day to 1 month.
#### Who is reviewing it? 

@Bdthomson @hayleynorton @lamhuy 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@bdeining
@djblue 
#### How should this be tested?

1. Build and install DDF
2. Click on the gear icon in top right of the search ui
3. Click on Notifications
4. Verify that persistent notifications are enabled and that the expiration date is 1 month.
#### What are the relevant tickets?
[DDF-4101](https://codice.atlassian.net/browse/DDF-4101)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/js/model/User.js [hunk 1]
```diff
@@ -127,8 +127,8 @@ define([
                 homeSort: 'Last modified',
                 homeDisplay: 'Grid',
                 alerts: [],
-                alertPersistance: false, // don't persist across sessions by default
-                alertExpiration: 86400000, // 1 day
+                alertPersistance: true, // persist across sessions by default
```
Comment (Schachte) [inline ui/packages/catalog-ui-search/src/main/webapp/js/model/User.js]: ✏️ persistence is spelled wrong here

**Comment by AzGoalie** [comment]
build now

**Comment by Bdthomson** [comment]
build now

**Comment by andrewzimmer** [comment]
Hero Successful 🎉
- Built/ran DDF
- Verified that persistent notifications are enabled and that the expiration date is set to 1 month

**Pull request merged**